docs-ref-conceptual/authenticate-azure-cli-managed-identity.md

```diff
@@ -22,11 +22,13 @@ Here are some of the benefits of using managed identities:
 
 Signing in with the resource's identity is done through the `--identity` flag.
 
+To sign in with a system-assigned managed identity:
+
 ```azurecli-interactive
 az login --identity
 ```
 
-If the resource has multiple user assigned managed identities and no system assigned identity, you must specify the client ID or object ID or resource ID of the user assigned managed identity with `--username` for login.
```

Comment:
1. Even if the resource has only one user-assigned managed identity, an ID should be specified.
2. This is not related to system-assigned managed identity. Even if the resource has a system-assigned managed identity, we can still use the user-assigned managed identity by specifying an ID.